packages/testkit/tsconfig.json

```diff
@@ -1,28 +1,22 @@
 {
-    "compilerOptions": {
-      "module": "commonjs",
-      "target": "es6",
-      "outDir": "dist",
-      "strict": true,
-      "sourceMap": true,
-      "strictNullChecks": true,
-      "moduleResolution": "node",
-      "importHelpers": true,
-      "esModuleInterop": true,
-      "skipLibCheck": true,
-      "forceConsistentCasingInFileNames": true,
-      "noUnusedLocals": true,
-      "noUnusedParameters": true,
-      "noFallthroughCasesInSwitch": true,
-      "strictBindCallApply": true,
-      "strictFunctionTypes": true,
-      "strictPropertyInitialization": true,
-      "rootDir": ".",
-      "declaration": false,
-      "downlevelIteration": true,
-      "experimentalDecorators": true,
-      "noImplicitOverride": true,
-      "noImplicitReturns": true,
-    },
-    "include": [ "buttons", "inputs", "navigation", "pickers", "popover", "text", "utils", "BaseElement.ts", "index.ts"]
-  }
+  "compilerOptions": {
+    "lib": ["ESNext"],
```

Comment:
Don't you need also "dom" in here?

Reply:
for e2e in general I believe you do
I'm not sure for testkit but I think so as well

Reply:
No we wouldn't need "dom" because  the DOM interaction is abstracted by testkit itself. Playwright directly handles actions like clicking elements, asserting visibility, or navigating pages, without needing direct access to browser-native APIs.